redhat-cop/infra-ansible
# Dns keys

### What does this PR do?
Enhancing the dns-server role to avoid updating nsupdate keys unnecessary.

### How should this be tested?
Run the test (in the test dir); then update/add additional views/zones to the test and re-run. Observe that the already existing keys were not updated.

### Is there a relevant Issue open for this?
N/A

### People to notify
cc: @day4skiing
